src/operation.ts

```diff
@@ -16,9 +16,9 @@ export enum Operators {
 
 export class Operation {
   private _args: any;
-  private _operator: Operators;
+  private _operator: Operators | string;
```

Comment:
Let's rename `Operators` to `Operator`

Reply:
I agree, but this doesn't have anything with custom operators. I suggest to do this in a separate pull request after this one is merged.